crates/stdsimd-test/src/lib.rs

```diff
@@ -154,7 +154,9 @@ pub fn assert(fnptr: usize, fnname: &str, expected: &str) {
                 // Original limit was 20 instructions, but ARM DSP Intrinsics
                 // are exactly 20 instructions long. So, bump the limit to 22
                 // instead of adding here a long list of exceptions.
-                _ => 22,
+		// Some MIPS MSA Intrinsics are 22 instructions long,
+		// so, previous limit of 22 is increased to 23.
+                _ => 23,
```

Comment:
@gnzlbg This limit of 23 instructions suits for the mipsisa64r6el target.
For mipsisa64r6 target, there are 98 intrinsics that are longer than this limit.
The length of these intrinsic is as follows:

```
10 intrinsics are 24 instructions long
54 intrinsics are 25 instructions long
24 intrinsics are 26 instructions long
4 intrinsics are 28 instructions long
6 intrinsics are 30 instructions long
```
Is it ok to just increase this limit to 31 or it is better to add a list of exceptions for these intrinsics?

Reply:
The exceptions are here: https://github.com/rust-lang-nursery/stdsimd/blob/master/crates/stdsimd-test/src/lib.rs#L153

You can just add a new match arm there, that does something like:

```rust
 _ if target.contains("mips") => 31,
```

and make this exception specific for mips only.

Reply:
I used `fnname` and this solves my problem:

`_ if fnname.contains("msa") => 31,`